pkgs/applications/version-management/git-and-tools/gh/default.nix

```diff
@@ -27,6 +27,18 @@ buildGoModule rec {
     done
   '';
 
+  # should be in the next release
+  # https://github.com/cli/cli/issues/401#issuecomment-586918622
+  # https://github.com/cli/cli/issues/401#issuecomment-588261787
+  patches = [
+    (
+      fetchpatch {
+        url = "https://github.com/cli/cli/commit/213f4a5333c21020da030012979bd326a34fb28c.patch";
+        sha256 = "0sp52khbbq8cfxgqs77qzxrmi7zwb12dnqps5z8sk063y30i97zp";
+      }
+    )
+  ];
```

Comment:
are you using nixpkgs-fmt? This format is actually a bug
```
patches = [
  (fetchpatch {
    url = ...
    sha256 = ...
  })
];

Reply:
Fixed.